### ISSUE TITLE
想问一下为什么我自己的数据 只有两列,会出现Nan这种情况

### ISSUE THREAD

**Issue by toby911** [thuml/Time-Series-Library#315]
Epoch: 50, Steps: 3 | Train Loss: 1.3646304 Vali Loss: nan Test Loss: 1.4440242
Validation loss decreased (nan --> nan).  Saving model ...
Updating learning rate to 1.7763568394002506e-20

**Comment by toby911** [comment]
用的是timesnet的long-pre,

**Comment by wuhaixu2016** [comment]
您好，我觉得可能需要检查一下有没有经过pre-norm，具体需要根据您的数据来调整normalization的方式。
